tools/dispersion.go

```diff
@@ -469,9 +470,9 @@ func (d *Dispersion) scanDispersionObjs(cancelChan chan struct{}) {
 }
 
 func PrintLastDispersionReport(serverconf conf.Config) error {
-	sqlDir, ok := serverconf.Get("drive_watch", "sql_dir")
-	if !ok {
-		return fmt.Errorf("Invalid Config, no drive_watch sql_dir")
+	sqlDir := serverconf.GetDefault("drive_watch", "sql_dir", "/var/local/hummingbird")
+	if _, err := os.Open(sqlDir); err != nil {
```

Comment:
I don't think it's a big deal, unless this runs a lot, but fd leak here right?
Maybe just better to leave the error on the sql.Open to handle if the dir is missing?